electrum/simple_config.py

```diff
@@ -1048,11 +673,7 @@ def __setattr__(self, name, value):
     TEST_SHUTDOWN_FEE_RANGE = ConfigVar('test_shutdown_fee_range', default=None)
     TEST_SHUTDOWN_LEGACY = ConfigVar('test_shutdown_legacy', default=False, type_=bool)
 
-    FEE_EST_DYNAMIC = ConfigVar('dynamic_fees', default=True, type_=bool)
```

Comment:
This is still used in `test_lnpeer.TestPeerDirect.test_close_upfront_shutdown_script`.

```
        w1.network.config.FEE_EST_DYNAMIC = False
        w2.network.config.FEE_EST_DYNAMIC = False
        w1.network.config.FEE_EST_STATIC_FEERATE = 5000
        w2.network.config.FEE_EST_STATIC_FEERATE = 1000
```
Unfortunately the ConfigVar logic is not smart enough to error on this :/  I wonder if I could fix that.

Reply:
ref https://github.com/spesmilo/electrum/pull/9600